tests/e2e-tests/src/orchestration.test.ts

```diff
@@ -1,34 +1,50 @@
-import { OrchestrationClient } from '@sap-ai-sdk/orchestration';
+import {
+  orchestrationChatCompletion,
+  orchestrationTemplating,
+  orchestrationInputFiltering,
+  orchestrationOutputFiltering,
+  orchestrationRequestConfig
+} from '@sap-ai-sdk/sample-code';
+import { OrchestrationResponse } from '@sap-ai-sdk/orchestration';
 import { loadEnv } from './utils/load-env.js';
 
 loadEnv();
 
 describe('orchestration', () => {
-  it('should complete a chat', async () => {
-    const response = await new OrchestrationClient({
-      llm: {
-        model_name: 'gpt-35-turbo-16k',
-        model_params: { max_tokens: 50, temperature: 0.1 }
-      },
-      templating: {
-        template: [
-          {
-            role: 'user',
-            content: 'Create {{?number}} paraphrases of {{?phrase1}}'
-          }
-        ]
-      }
-    }).chatCompletion({
-      inputParams: {
-        number: '3',
-        phrase1: 'I love coffee'
-      }
-    });
-
+  const assertContent = (response: OrchestrationResponse) => {
```

Comment:
Oh, ok, that is why you moved the `getContent()`. Oh well, consider my comments anyways, maybe you have an idea how to combine this, if not this was `[pp]` anyways ;)

Reply:
Thanks! This was also something disturbing me for quite sometime since I had the idea to integrate zod into e2e test. In the future, we can use zod to check response instead of manually assert each and every attributes. Maybe we want to change other sample code as well to always return response and log content.